mimic/model/nova_objects.py

```diff
@@ -527,6 +527,35 @@ def fail_later(server):
     return fail_later
 
 
+@server_creation.declare_behavior_creator("sequence")
+def sequence(parameters):
+    """
+    Sometimes a sequence of behaviors occur when you try to create a server in
+    a predictable pattern.
+
+    Takes one parameter, ``behaviors``, which is a list of specifications of
```

Comment:
Fundamentally you're right, this documentation is indecipherable, I will include an example and describe it more clearly.